src/PageObject/AdminPage.php

```diff
@@ -78,7 +78,14 @@ protected function getHeaderElement()
      */
     public function clickLinkInHeader($link)
     {
+        $headerLink = $this->find('css', '.page-title-action');
```

Comment:
PR looks fine but please can we use something like `header_link` instead of `headerLink` for variable names? Only doing the caps thing for function or method names.